README.md

```diff
@@ -381,3 +173,6 @@ There are lots of mach commands you can use. You can list them with `./mach
 
 
 The generated documentation can be found on https://doc.servo.org/servo/index.html
+
+[manual-build]: https://github.com/servo/servo/wiki/Building#manual-build-setup
+[windows-tips]: https://github.com/servo/servo/wiki/Building#troubleshooting-the-windows-build
```

Comment:
I don't find how to leave a comment on an unmodified line, but some lines earlier we have:
```sh
./mach build --dev
./mach run tests/html/about-mozilla.html
```

It'd be nice to add `--dev` on `mach run` too.

Reply:
The rest of the README should be cleaned up as well, but I wanted to focus on the build instructions for the first pass.

Reply:
Ok, this looked so small that we could just do it, but we can do it later too.